tests/dryrun/assets/cluster-config.yaml

```diff
@@ -8,20 +8,18 @@ spec:
    replicatedAppDomain: "fake-endpoint.com"
    proxyRegistryDomain: "fake-replicated-proxy.test.net"
  unsupportedOverrides:
-   builtInExtensions:
```

Comment:
this requires more changes to types than i thought, and is already covered by all of our e2e tests